src/pages/AboutBase/FormAddSubsection.tsx

```diff
@@ -22,6 +22,13 @@ type FormAddSubsectionProps = {
     fetchSubData: Function
 };
 
+const normFile = (e: any) => {
```

Comment:
what does it mean, normFile?

and can't we just use some type, instead of any?

Reply:
This feature adds files to the download list. To a function passed a large object, so "any" is required.